tests/unit/systems/ops/tf/test_ensemble.py

```diff
@@ -111,6 +112,81 @@ def test_workflow_tf_e2e_config_verification(tmpdir, dataset, engine):
     assert len(response["output"]) == df.shape[0]
 
 
+def raise_(col):
+    if (
+        isinstance(col.dtype, (type(numpy.dtype("float64")), type(numpy.dtype("int64"))))
+        and col.sum() != 0
+    ):
+        return col
+    else:
+        raise ValueError("Number Too High!!")
+
+
+@pytest.mark.skipif(not TRITON_SERVER_PATH, reason="triton server not found")
+@pytest.mark.parametrize("engine", ["parquet"])
+def test_workflow_tf_e2e_error_propagation(tmpdir, dataset, engine):
+    # Create a Workflow
+    schema = dataset.schema
+    for name in ["x", "y", "id"]:
+        dataset.schema.column_schemas[name] = dataset.schema.column_schemas[name].with_tags(
+            [Tags.USER]
+        )
+    selector = ColumnSelector(["x", "y", "id"])
+
+    workflow_ops = selector >> wf_ops.Rename(postfix="_nvt") >> wf_ops.LambdaOp(raise_)
+    workflow = Workflow(workflow_ops["x_nvt"])
+    workflow.fit(dataset)
+
+    # Create Tensorflow Model
```

Comment:
I think you could simplify this test a bit by dropping the model and making a one op ensemble